src/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Game Leaderboard Projects</title>
+  </head>
+  <body class="container">
+    <header class="mt-5">
+      <h1 class="heading">Game Leaderboard</h1>
+    </header>
+    <main class="container d-flex flex-row justify-content-between mt-5">
+      <section class="col-6">
+        <div class="d-flex flex-row justify-content-between">
+          <h2>Recent Scores</h2>
+          <button class="btn btn-outline-secondary">Refresh</button>
+        </div>
+        <ul class="leaderBoard"></ul>
+      </section>
+      <section class="col-4">
+        <h2>Add your score</h2>
+        <form class="d-flex flex-column justify-content-between">
+          <input class="name" type="text" placeholder="Your name" />
+          <input class="score" type="text" placeholder="Your score" />
+          <div>
+            <button id="addScores" class="btn btn-outline-secondary">
+              Submit
+            </button>
+          </div>
+        </form>
+      </section>
```

Comment:
- [ ] Even though styling is not a requirement, we have to follow the wireframe provided as it guides us in laying out the elements on our page.

Your design
![image](https://github.com/Errytagedesign/gameLeaderboard/assets/81932478/bd7d520d-3147-4875-9e44-7243e48050da)

Given wireframe
![image](https://github.com/Errytagedesign/gameLeaderboard/assets/81932478/0c983629-0bd0-493f-96e6-9005b5f15ad6)

- kindly ensure your `Submit` button is aligned as in the given wireframe.
- Also consider adding a `<br/>` line break to separate the two input fields and the button.